tests/fips/mozilla_nss/firefox_nss.pm

```diff
@@ -29,6 +29,37 @@ sub quit_firefox {
     }
 }
 
+sub firefox_crashreporter {
+
+    select_console 'root-console';
+
+    my $firefox_crash_dir = '/home/bernhard/.mozilla/firefox/Crash\ Reports';
+    my $crash_report      = '/home/bernhard/.mozilla/firefox/Crash\ Reports/crashreporter.ini';
+
+    assert_script_run("ls $firefox_crash_dir");
+
+    assert_script_run("sed -i -e 's/EmailMe=0/EmailMe=1/g' $crash_report");
+    assert_script_run("sed -i -e 's/SubmitReport=0/SubmitReport=1/g' $crash_report");
+    assert_script_run "cat $crash_report";
+    assert_script_run "cat $crash_report | grep 'EmailMe=1'";
+    assert_script_run "cat $crash_report | grep 'SubmitReport=1'";
```

Comment:
Ben, can you please use the same code style for assert_script_run()?

Reply:
SURE, you are definitely right :)

Reply:
Resolved.